docs/contributing/dev-setup/index.rst

```diff
@@ -3,20 +3,42 @@
 Developer Setup
 ===============
 
-To ease developer setup, we have Ansible roles, playbooks and a Vagrantfile in our `pulp/devel
-<https://github.com/pulp/devel/>`_ repository. Use the :ref:`quickstart` to dive in with an example
-configuration.
+To ease developer setup, we have `Pulplift <https://github.com/ehelms/pulplift>`_ which is based on
```

Comment:
Does it make any sense to move this into the Pulp or and update this link to that pre-merge?

Reply:
Let's update the link after the repo is transferred.